src/modules/ie-objects/ObjectDetailPage.tsx

```diff
@@ -1413,7 +1413,11 @@ export const ObjectDetailPage: FC<DefaultSeoInfo> = ({ title, description, image
 	};
 
 	const renderOcrBlade = () => {
-		if (!isOpenSeaDragonPoc || !OPEN_SEA_DRAGON_POC_IMAGE_INFOS[activeImageIndex].altoUrl) {
+		if (
+			!isOpenSeaDragonPoc ||
+			!OPEN_SEA_DRAGON_POC_IMAGE_INFOS[activeImageIndex].altoUrl ||
+			(activeTab === ObjectDetailTabs.Media && isMobile)
```

Comment:
Hide elements responsively using css, not js. This keeps display rules centralised and avoids tearing your hair out during debugging

Reply:
This ocr drawer will be moved according to "yet to be approved design"
then it will be inline in de metadata sidebar, so this will become a non issue

i will leave it as is, until the new design is approved, then make sure the isMobile variable isn't used for this anymore

![image](https://github.com/viaacode/hetarchief-client/assets/1710840/6d8dc56a-4b80-4914-a4e6-1761e3de8c65)